### PR TITLE
Clarify what happens when a concern is raised during FCP

### PR DESCRIPTION
**Pull request by anoadragon453**
It wasn't entirely clear what should happen to the FCP timer (and state) when a concern is raised during FCP. After some discussion, we agreed that when a concern is raised:

1. FCP will continue to not conclude until at least 5 days have passed, but once those 5 days are up it *still* won't conclude until all concerns raised during FCP are resolved.
2. If a concern warrants a large enough change in the document, then the Spec Core Team may consider cancelling FCP and restarting the timer in order for people to have some time to think about and review the new changes.

### CODE REVIEW

**Review by richvdh — APPROVED** [review]
lgtm.

File: content/proposals.md [hunk 1]
```diff
@@ -245,10 +245,13 @@ is as follows:
...
-    time to speak up before it concludes. On its conclusion, the
-    disposition of the FCP will be carried out. If sufficient reasoning
-    against the disposition is raised, the FCP can be cancelled and the
-    MSC will continue to evolve accordingly.
+    time to speak up before it concludes. If sufficient reasoning
+    against the disposition is provided, a member of the Spec Core Team can
+    raise a concern and block FCP from completing. This will not reset or
+    pause the 5 day FCP timer, but FCP will not conclude until all concerns have
+    been resolved. If sufficient change in the MSC is required to resolve those
+    concerns, FCP may be cancelled and reproposed. Once FCP has concluded,
```
Comment (turt2live) [inline content/proposals.md:253]: As part of my quest to remove "may" from the spec:
```suggestion
    concerns, FCP might be cancelled and reproposed. Once FCP has concluded,
```

Comment (anoadragon453) [reply content/proposals.md:253]: Oh, I write "may" all the time. Should I not be? :D

Comment (turt2live) [reply content/proposals.md]: in the spec documents, preferably not. It's confusing as a word to many non-native English speakers, so I've been trying to use the replacements `could` and `might` where needed.

**Pull request merged**